packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -491,21 +503,74 @@ To edit platform code in an IDE see https://flutter.dev/developing-packages/#edi
 void _printPluginUpdatePubspecMessage(String pluginPath, String platformsString) {
   globals.printStatus('''
 You need to update $pluginPath/pubspec.yaml to support $platformsString.
-
 ''', emphasis: true, color: TerminalColor.red);
 }
 
 void _printNoPluginMessage() {
     globals.printError('''
 You've created a plugin project that doesn't yet support any platforms.
-
 ''');
 }
 
-void _printAddPlatformMessage(String pluginPath) {
+void _printPluginAddPlatformMessage(String pluginPath) {
   globals.printStatus('''
 To add platforms, run `flutter create -t plugin --platforms <platforms> .` under $pluginPath.
 For more information, see https://flutter.dev/go/plugin-platforms.
 
 ''');
 }
+
+/// returns a list of platforms which user has requested a platform and if it's not enabled
+List<String> _getPlatformWarningList(List<String> requestedPlatforms) {
+  final List<String> _disabledPlatform = <String>[
+  if (requestedPlatforms.contains('android') && !featureFlags.isAndroidEnabled)
+    'android',
+  if (requestedPlatforms.contains('ios') && !featureFlags.isIOSEnabled)
+    'ios',
+  if (requestedPlatforms.contains('web') && !featureFlags.isWebEnabled)
+    'web',
+  if (requestedPlatforms.contains('macos') && !featureFlags.isMacOSEnabled)
+    'macos',
+  if (requestedPlatforms.contains('windows') && !featureFlags.isWindowsEnabled)
+    'windows',
+  if (requestedPlatforms.contains('linux') && !featureFlags.isLinuxEnabled)
+    'linux',
+  ];
+
+  return _disabledPlatform;
+}
+
+void _printWarningDisabledPlatform(List<String> platforms, String relativePath) {
+  final  List<String> _mobile = <String>[], _desktop = <String>[], _web = <String>[];
```

Comment:
use camel case for ivar.

Reply:
They are in camel case only. Do you want me to make them non-private var?

Reply:
Can you give me an example for it?

Reply:
Sorry, what I meant is not to use the _